virttest/qemu_devices/qcontainer.py

```diff
@@ -1019,6 +1019,10 @@ def pflash_handler(firmware_name, cmd):
             """
             machine_cmd = cmd
             devs = []
+            images = params.objects('images')
+            if not images:
+                return devs, machine_cmd
+
```

Comment:
```suggestion
```

Reply:
Is there any difference in this suggestion?